pkgs/games/cataclysm-dda/stable.nix

```
@@ -18,49 +17,21 @@ let
   };
 
   self = common.overrideAttrs (common: rec {
-    version = "0.G";
+    version = "0.H";
 
     src = fetchFromGitHub {
       owner = "CleverRaven";
       repo = "Cataclysm-DDA";
-      rev = version;
-      sha256 = "sha256-Hda0dVVHNeZ8MV5CaCbSpdOCG2iqQEEmXdh16vwIBXk=";
+      rev = "${version}-RELEASE";
```

```suggestion
      tag = "${version}-RELEASE";
```
Recently introduced.

---

Please don't give stylistic changes with requesting change button. This is not blocking.